libs/ui-lib/lib/common/components/ui/ExternalLink.tsx

```diff
@@ -1,15 +1,24 @@
-import React, { ReactNode } from 'react';
+import React, { ReactNode, MouseEvent } from 'react';
 import { Text } from '@patternfly/react-core';
 import { ExternalLinkAltIcon } from '@patternfly/react-icons';
 
 interface ExternalLinkProps {
   href?: string;
-  onClick?: VoidFunction;
+  onClick?: (event: MouseEvent<HTMLAnchorElement>) => void;
   children?: ReactNode;
 }
-const ExternalLink: React.FC<ExternalLinkProps> = ({ href, children, onClick }) => (
-  <Text component="a" href={href} onClick={onClick} target="_blank" rel="noopener noreferrer">
-    {children ? children : href} <ExternalLinkAltIcon color="rgb(0, 123, 186)" />
-  </Text>
-);
+const ExternalLink: React.FC<ExternalLinkProps> = ({ href, onClick, children }) => {
+  const handleClick = (event: MouseEvent<HTMLAnchorElement>) => {
+    event.stopPropagation(); // Stop event propagation here
```

Comment:
We don't need to remove the e.stopPropagation in PreviewBadge. This change was done here https://github.com/openshift-assisted/assisted-installer-ui/pull/1960 to prevent extension of click event.